task_handler_layer/task_buoy/CMakeLists.txt

```diff
@@ -89,6 +92,9 @@ find_package( OpenCV REQUIRED )
 #   cfg/DynReconf1.cfg
 #   cfg/DynReconf2.cfg
 # )
+add_message_files(DIRECTORY msg
+  FILES ResizedImage.msg
```

Comment:
are u using this ? maybe u can remove this msg file from here and the msg folder itself. Also what is this resized_plugins.xml in include folder. I think these are left over folders, both `msg` and `include`